### PR TITLE
Fix/terminal-icons

### PR DESCRIPTION
**Pull request by hansnet**

### CODE REVIEW

File: src/docker-compose.override.yml [hunk 1]
```diff
@@ -6,13 +6,13 @@ services:
       - ./var1.env
     environment:
       - ASPNETCORE_ENVIRONMENT=Development
-      - ASPNETCORE_URLS=https://+:65103;http://+:80
+      - ASPNETCORE_URLS=https://+:64103;http://+:80
```
Comment (elmerbool) [inline src/docker-compose.override.yml:9]: Please revert your changes to docker-compose.override.yml.

File: src/main/Port.Adapter/UI/Views/Blazor/Shared/EditorBox.razor [hunk 2]
```diff
@@ -59,6 +59,31 @@
                                     <ConditionalDisplay Show="@true">
                                         <li style="list-style:none" @key="@itemContext.Id">
                                             <input type="radio" id="@itemContext.Id" checked="@(this.EditTerminal != null && this.EditTerminal.Id == itemContext.Id)" @onchange="@(() => { this.EditTerminal = itemContext; })" />
+                                            @if (itemContext.Neuron.Type != Library.Common.RelativeType.NotSet)
```
Comment (elmerbool) [inline src/main/Port.Adapter/UI/Views/Blazor/Shared/EditorBox.razor:62]: 1. Are you able to move the logic below into the method GetAttribute? (See RenderFragment line 125 of BlazoriseSidebar.razor)
2. Then rename GetAttribute to GetSynapticSymbolSpan(itemContext.Neuron), and then evaluate strength, effect, and type in the method itself, then return the appropriate span RenderFragment.

**Review by elmerbool — CHANGES_REQUESTED** [review]

File: src/docker-compose.override.yml (hunk 1 above)
Comment (hansnet) [reply src/docker-compose.override.yml]: fixed this

File: src/main/Port.Adapter/UI/Views/Blazor/Shared/EditorBox.razor (hunk 2 above)
Comment (hansnet) [reply src/main/Port.Adapter/UI/Views/Blazor/Shared/EditorBox.razor]: moved the logic and renamed the method

**Comment by hansnet** [comment]
Changes were made on a new branch

**Pull request closed without merging**